en/lessons/basics/control-structures.md

```diff
@@ -1,207 +1,246 @@
 ---
-version: 1.1.1
+version: 1.0.0
 title: Control Structures
 ---
 
-In this lesson we will look at the control structures available to us in Elixir.
+Recall that Haskell functions are *definitions*, not lists of instructions
 
-{% include toc.html %}
+In imperative programming, control structures are used to decide which block of
+code should be executed
 
-## `if` and `unless`
+In Haskell, they are used instead to decide how a certain value should be
+defined
 
-Chances are you've encountered `if/2` before, and if you've used Ruby you're familiar with `unless/2`.
-In Elixir they work much the same way but they are defined as macros, not language constructs. You can find their implementation in the [Kernel module](https://hexdocs.pm/elixir/Kernel.html).
+__Note__: Usually control structures are used directly in the definition of a 
+function, but they can also be used in the definition of any intermediate
+value inside a function (we will see examples of both)
 
-It should be noted that in Elixir, the only falsey values are `nil` and the boolean `false`.
+Let's have a look at the control structures available in Haskell, and some
+examples of how they work
 
-```elixir
-iex> if String.valid?("Hello") do
-...>   "Valid string!"
-...> else
-...>   "Invalid string."
-...> end
-"Valid string!"
+{% include toc.html %}
 
-iex> if "a string value" do
-...>   "Truthy"
-...> end
-"Truthy"
-```
+## Extensions and Imports
 
-Using `unless/2` is like `if/2` only it works on the negative:
+You will need to add the following lines to the top of your Haskell source 
+file (`.hs`) to be able to run these examples
 
-```elixir
-iex> unless is_integer("hello") do
-...>   "Not an Int"
-...> end
-"Not an Int"
+```haskell
+{-# LANGUAGE OverloadedStrings #-}
+
+import Data.Text ( Text )
 ```
 
-## `case`
+## If Statements
+
+Haskell provides a simple set of keywords for `if` statements, with only a 
+single `else` condition
 
-If it's necessary to match against multiple patterns we can use `case/2`:
+### Syntax
 
-```elixir
-iex> case {:ok, "Hello World"} do
-...>   {:ok, result} -> result
-...>   {:error} -> "Uh oh!"
-...>   _ -> "Catch all"
-...> end
-"Hello World"
+```
+result = if (boolExpression) 
+         then (expression1)
+         else (expression2)
 ```
 
-The `_` variable is an important inclusion in `case/2` statements. Without it, failure to find a match will raise an error:
+- If the first expression evaluates to `True`, then the expression after the
```

Comment:
Would it be overdoing things to recall the users with a background in dynamic languages that there is no type coercion at this level, so that `True` really a value constructed from the 0-place `True` constructor and not some "truthy" value as in Javascript or Python?

Reply:
Booleans are already covered in the first chapter: https://github.com/haskellfoundation/HaskellSchool/blob/4df5d05991e720cbc9255685a45388ed8dcf1900/en/lessons/basics/basics.md#booleans